source/agora/consensus/ValidatorSet.d

```diff
@@ -108,13 +108,15 @@ public class ValidatorSet
 
         try
         {
-            static ubyte[] buffer;
+            static ubyte[] buffer, pbuffer;
```

Comment:
I prefer moving variable declarations as close to where they're initialized as possible. So just above the `serializeToBuffer(img, pbuffer);` call. 😇

Reply:
Variables reorganized my good sir!